mesheryctl/internal/cli/root/root.go

```diff
@@ -61,10 +60,14 @@ mesheryctl -v [or] --verbose
 	`,
 	// Uncomment the following line if your bare application
 	// has an action associated with it:
-	PreRunE: func(cmd *cobra.Command, args []string) error {
-		log.Println("Args passed in", args)
+
+	Args: cobra.MaximumNArgs(1),
```

Comment:
You should remove this line. Because now, here is the behaviour if there's a slight misspell in a large command.

```
➜ mesheryctl syste context view --all
 
Error: accepts at most 1 arg(s), received 2
```

Reply:
This is not resolved?

Reply:
@harkiratsm

Reply:
If I remove that then It would be handled by the ```cobra``` itself , which is good and bad at same time. 
```bash
-> mesheryctl syste context
Error: unknown command "syste" for "mesheryctl"

Did you mean this?
        system

Run 'mesheryctl --help' for usage.
```
```bash 
-> mesheryctl asdsd conte
Error: unknown command "asdsd" for "mesheryctl"
Run 'mesheryctl --help' for usage.
```

Reply:
@nishantwrp

Reply:
@harkiratsm totally missed this comment 🤦‍♂️. Yes removing it would have better behaviour than it has now.

Reply:
@nishantwrp Ok